client/src/client.rs

```diff
@@ -196,31 +268,46 @@ impl Client {
         &self,
         transaction: Transaction,
     ) -> Result<HashOf<VersionedTransaction>> {
+        let (req, hash, resp_handler): (DefaultRequestBuilder<_>, _, _) =
+            self.prepare_transaction_request(transaction)?;
```

Comment:
```suggestion
        let (req, hash, resp_handler) =
            self.prepare_transaction_request::<DefaultRequestBuilder<_>>(transaction)?;
```

Reply:
Ok, will prefer this syntax. Is it better because of reducing of compiler's work? I heard that "turbofish" uglifies code.

Reply:
It doesn't really reduce the amount of work, because you either backtrack to infer the type, or propagate the type forward. What this does is remove the need for two anonymous types.

Reply:
> I heard that "turbofish" uglifies code.

I haven't heard that...

https://turbo.fish/